WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartNoticeDetails.kt

```diff
@@ -0,0 +1,76 @@
+package org.wordpress.android.ui.quickstart
+
+import org.wordpress.android.R.string
+import org.wordpress.android.fluxc.store.QuickStartStore
+import org.wordpress.android.fluxc.store.QuickStartStore.QuickStartTask
+
+/**
+ * Static data that represents info that goes into Quick Start notices
+ */
+enum class QuickStartNoticeDetails(
+    val taskString: String,
+    val titleResId: Int,
+    val messageResId: Int
+) {
```

Comment:
Nitpick: @ashiagr Thank you for converting this to Kotlin. I think adding the Resource annotation(@StringRes) to the variables here would be a good improvement. WDYT?  🤔

Reply:
Agree, added in 360aec2 for `QuickStartMySitePrompts`, `QuickStartNoticeDetails`, `QuickStartTaskDetails`.